### PR TITLE
QRadar race conditions

### PR DESCRIPTION
**Pull request by tomneeman151293**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/38284
## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 5.5.0
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

**Comment by tomneeman151293** [comment]
@dantavori Let me know when I can send it to the customer to check for no duplications.

**Comment by tomneeman151293** [comment]
@dantavori I think we can merge this. let me know for CR comments

File: Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py [hunk 1]
```diff
@@ -1510,21 +1510,19 @@ def long_running_execution_command(client: Client, params: Dict):
...
             )
-            # Reset was called during execution, skip creating incidents.
-            if not incidents and not new_highest_id:
-                continue
+            if incidents and new_highest_id:
+                incident_batch_for_sample = incidents[:SAMPLE_SIZE] if incidents else json.loads(
+                    ctx.get('samples', '[]'))
+                if incident_batch_for_sample:
+                    print_debug_msg(f'Saving New Highest ID: {new_highest_id}')
+                    set_to_integration_context_with_retries({'samples': incident_batch_for_sample,
```
[the remark this answers is not in the document]
Comment (dantavori) [reply Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py:1522]: Yes

[the remark this answers is not in the document]
Comment (tomneeman151293) [reply Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py:1522]: Nope. it is expected to be a dict, see 
`set_to_integration_context_with_retries` calls to
`update_integration_context`

File: Packs/QRadar/ReleaseNotes/2_0_27.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+
+#### Integrations
+##### IBM QRadar v3
+- Fixed an issue where configured **Long Running Instance** along with **Mirror Offenses** fetched duplicate incidents.
```
Comment (amshamah419) [inline Packs/QRadar/ReleaseNotes/2_0_27.md:4]:
```suggestion
- Fixed an issue where both **Long Running Instance** and **Mirror Offenses** would fetch duplicate incidents. 
```

**Pull request merged**